### ISSUE TITLE
.progress.active not animating

### ISSUE THREAD

**Issue by aaronsevers** [twbs/bootstrap-sass#616]
Active progress bars are not animating in my Rails 4 app with bootstrap-sass 3.1.1.1. Downgrading to 3.1.1.0 fixes the issue.

My code:

```
<div class="progress progress-striped progress-small active">
  <div class="progress-bar progress-bar-success" style="width: 100%"></div>
</div>
```

**Comment by leomayleomay** [comment]
it's fixed in v3.2.0, but the latest support for Rails 3.2 is v3.1.1.0, I am wondering what's the fix for the projects still using Rails 3.2

**Comment by yassermog** [comment]
add this to your css ... it worked for me 
.active{
   animation: progress-bar-stripes 2s linear infinite;
}
